src/Controller/OrderMessageController.php

```diff
@@ -0,0 +1,122 @@
+class OrderMessageController
+{
+    /** @var TranslatorInterface */
+    private $translator;
+    /** @var EngineInterface */
+    private $templatingEngine;
+    /** @var EntityManagerInterface */
+    private $entityManager;
+    /** @var OrderRepositoryInterface */
+    private $orderRepository;
+    /** @var SenderInterface */
+    private $mailer;
+    /** @var RouterInterface */
+    private $router;
+    /** @var FlashBagInterface */
+    private $flashBag;
+    /** @var FormFactoryInterface */
+    private $builder;
+    /** @var TokenStorageInterface */
+    private $token;
+
+    public function __construct(
+        TranslatorInterface $translator,
+        EngineInterface $templatingEngine,
+        EntityManagerInterface $entityManager,
+        OrderRepositoryInterface $orderRepository,
+        SenderInterface $mailer,
+        RouterInterface $router,
+        FlashBagInterface $flashBag,
+        FormFactoryInterface $builder,
+        TokenStorageInterface $tokenStorage
+    ) {
+        $this->translator = $translator;
+        $this->templatingEngine = $templatingEngine;
+        $this->entityManager = $entityManager;
+        $this->orderRepository = $orderRepository;
+        $this->mailer = $mailer;
+        $this->router = $router;
+        $this->flashBag = $flashBag;
+        $this->builder = $builder;
+        $this->token = $tokenStorage;
+    }
+
+    public function contactAction(Request $request, int $id): Response
+    {
+        $contact = new OrderMessage();
+        $form = $this->builder->create(OrderMessageType::class, $contact, [
+            'action' => $this->router->generate('mango_sylius_admin_order_message_send', ['id' => $id]),
+            'method' => 'POST',
+        ]);
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted()) {
+            if ($form->isValid()) {
+                $order = $this->orderRepository->find($id);
+                $token = $this->token->getToken();
+                $sendMail = $form['sendMail']->getData();
+                $contact->setSendTime(new \DateTime());
+                assert($order instanceof OrderInterface);
+                $contact->setOrder($order);
+
+                assert($token && $token->getUser() instanceof AdminUser);
+                $sender = $token->getUser();
+                $contact->setSender($sender);
+
+                $this->entityManager->persist($contact);
+                $this->entityManager->flush();
+                $customer = $order->getCustomer();
+                if ($sendMail === true) {
+                    assert($customer instanceof CustomerInterface);
+                    $this->mailer->send('order_mail', [$customer->getEmail()], ['contact' => $contact]);
+                    $this->flashBag->add('success', $this->translator->trans('mango_sylius.orderMessage.success.mail'));
+                } else {
+                    $this->mailer->send('note_mail', [$sender->getEmail()], ['contact' => $contact]);
```

Comment:
Remove this function, template and config. 

The administrator on the order details and knows that he created a note, there is no need to send an email. It is not the task.

Reply:
DONE

Reply:
DONE